frontends/web/src/routes/account/send/send-wrapper.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useContext } from 'react';
+import { IAccount } from '../../../api/account';
+import { TDevices } from '../../../api/devices';
+import RatesContext from '../../../contexts/RatesContext';
+import { Send } from './send';
+
+type TSendProps = {
+    accounts: IAccount[];
+    code: string;
+    devices: TDevices;
+    deviceIDs: string[];
+}
+
+export const SendWrapper = ({ accounts, code, deviceIDs, devices }: TSendProps) => {
+  const { defaultCurrency } = useContext(RatesContext);
+  return (
+    <Send
+      accounts={accounts}
+      code={code}
+      devices={devices}
+      deviceIDs={deviceIDs}
+      activeCurrency={defaultCurrency}
+    />
```

Comment:
just a comment: I guess here it could pass btcUnit, so that send doesn't have to fetch it from config.

but thinking about it, it's probably better if we wait and refactor send to functional component first and then remove the wrapper and use btcUnit from context.

Reply:
oh just read this after I wrote my comment above. Yeah seems like its the sme thing and we could totally pass it down here. I agree on just waiting for finish of func. component first.